src/components/signInForm.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+
+import { Formik, Field, Form, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+
+function SignInForm (props){
+    const onSubmit = props.onSubmit
+    const mutationError = props.mutationError
```

Comment:
No need for extra constants for these props.

```suggestion
function SignInForm ({ onSubmit, mutationError }){
```